components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/org.wso2.carbon.event.output.adapter.ui.endpoint/src/main/java/SuperTenantSubscriptionEndpoint.java

```diff
@@ -54,7 +53,7 @@ public void onOpen (Session session, @PathParam("streamname") String streamName
             log.debug("WebSocket opened, for Session id: "+session.getId()+", for the Stream:"+streamName);
         }
         PrivilegedCarbonContext carbonContext = PrivilegedCarbonContext.getThreadLocalCarbonContext();
-        carbonContext.setTenantId(-1234);
+        carbonContext.setTenantId(carbonContext.getTenantId());
         tenantId = carbonContext.getTenantId();
```

Comment:
Damith, I have an issue in above change.. If tenant id is there in the carbon context then why you are getting that and again setting that to carbon context.. What is expected here ?  I think there is an issue here.